client/my-sites/checkout/composite-checkout/components/contact-details-container.tsx

```diff
@@ -73,56 +87,42 @@ export default function ContactDetailsContainer( {
 					isDisabled={ isDisabled }
 					isLoggedOutCart={ isLoggedOutCart }
 				/>
-			</React.Fragment>
-		);
-	}
+			);
 
-	if ( isGSuiteInCart ) {
-		return (
-			<DomainContactDetails
-				domainNames={ domainNames }
-				contactDetails={ contactDetails }
-				contactDetailsErrors={ contactDetailsErrors }
-				updateDomainContactFields={ updateDomainContactFields }
-				shouldShowContactDetailsValidationErrors={ shouldShowContactDetailsValidationErrors }
-				isDisabled={ isDisabled }
-				isLoggedOutCart={ isLoggedOutCart }
-			/>
-		);
-	}
+		default:
+			return (
+				<React.Fragment>
+					<ContactDetailsFormDescription>
+						{ translate( 'Entering your billing information helps us prevent fraud.' ) }
+					</ContactDetailsFormDescription>
 
-	return (
-		<React.Fragment>
-			<ContactDetailsFormDescription>
-				{ translate( 'Entering your billing information helps us prevent fraud.' ) }
-			</ContactDetailsFormDescription>
-
-			{ isLoggedOutCart && (
-				<Field
-					id="email"
-					type="email"
-					label={ String( translate( 'Email' ) ) }
-					disabled={ isDisabled }
-					onChange={ ( value ) => {
-						updateEmail( value );
-					} }
-					autoComplete="email"
-					isError={ email.isTouched && ! isValid( email ) }
-					errorMessage={ email.errors[ 0 ] || '' }
-					description={ String(
-						translate( "You'll use this email address to access your account later" )
+					{ isLoggedOutCart && (
+						<Field
+							id="email"
+							type="email"
+							label={ String( translate( 'Email' ) ) }
+							disabled={ isDisabled }
+							onChange={ ( value ) => {
+								updateEmail( value );
+							} }
+							autoComplete="email"
+							isError={ email.isTouched && ! isValid( email ) }
+							errorMessage={ email.errors[ 0 ] || '' }
+							description={ String(
+								translate( "You'll use this email address to access your account later" )
+							) }
+						/>
 					) }
-				/>
-			) }
 
-			<TaxFields
-				section="contact"
-				taxInfo={ contactInfo }
-				updateCountryCode={ updateCountryCode }
-				updatePostalCode={ updatePostalCode }
-				countriesList={ countriesList }
-				isDisabled={ isDisabled }
-			/>
-		</React.Fragment>
-	);
+					<TaxFields
```

Comment:
Could you explain more about not differentiating between `'none'` and `'tax'` in the default case?

Reply:
In theory, this component should handle `'none'`, but it will never be rendered at all in that case, so it ignores that value.

Reply:
After exploring this some more, it looks like some of these shouldn't get a 'none' passed to them (they're behind `shouldShowContactStep` which is `contactDetailsType !== 'none'`). Should 'none' be excluded from `ContactDetailsContainer` and `WPContactForm`? `Exclude< ContactDetailsType, 'none' >`?